pkg/inject/inject.go

```diff
@@ -151,6 +169,14 @@ func (conf *ResourceConfig) AppendPodAnnotations(annotations map[string]string)
 	}
 }
 
+// CreatedByCLI returns true if the pod is annotated with linkerd.io/created-by: linkerd/cli
+func (conf *ResourceConfig) CreatedByCLI() bool {
+	if conf.pod.annotations == nil {
+		return false
+	}
+	return strings.Contains(conf.pod.annotations[k8s.CreatedByAnnotation], "")
```

Comment:
I believe this will return true iff ` conf.pod.annotations[k8s.CreatedByAnnotation]` is defined

Reply:
Ah..sorry, that's obsolete; will remove.